packages/api/src/external/hie/cross-hie-ids.ts

```diff
@@ -1,5 +1,5 @@
-import { getOrganizationIds } from "../carequality/command/cq-directory/cq-gateways";
+import { getOrganizationIdsByManagingOrgs } from "../carequality/command/cq-directory/cq-gateways";
 
 export async function getCqOrgIdsToDenyOnCw() {
-  return getOrganizationIds(["CommonWell"]);
+  return getOrganizationIdsByManagingOrgs(["CommonWell", "Particle Health"]);
```

Comment:
If this works as expected - shouldn't we then add all gateways? Like athena for example?

I thought this was already working as expected with Athena though?

Reply:
From my understanding this was not the case.

On [this PR](https://github.com/metriport/metriport/commit/39cf0672c355923944fba207fdfc69d718d43af3#diff-ed9bffa17b44ed55fd85ecc24fcd1e9cab83eb5f2d1e6904cdc906f116d50095L54) we moved from excluding all entries with XCPD URL to the ones with managing org = CW.

I remember we talking that filtering the ones w/ managing org = CW would do it.
Now that we have the CW directory we can better tell which ones are on both - https://github.com/metriport/metriport-internal/issues/1783